src/zmq/zmqnotificationinterface.cpp

```diff
@@ -100,7 +100,6 @@ bool CZMQNotificationInterface::Initialize()
 
     if (i!=notifiers.end())
     {
-        Shutdown();
```

Comment:
The destructor will call it, which will be invoked automatically whether Initialize returns false or not.